src/howe_lab_to_nwb/vu2024/metadata/vu2024_fiber_photometry_metadata.yaml

```diff
@@ -3,32 +3,41 @@ Ophys:
     FiberPhotometryTable:
        name: FiberPhotometryTable
        description: Contains the metadata for the fiber photometry experiment.
+    FiberPhotometryResponseSeries:
+      - name: FiberPhotometryResponseSeries
+        description: Raw fluorescence traces acquired with multi-fiber array implanted in the striatum.
+        indicator: GCaMP7f
+        optical_fiber: [Fiber34]
+        excitation_source: ExcitationSource470
+        dichroic_mirror: DichroicMirror1
+        excitation_filter: OpticalFilter473
+        photodetector: PhotodetectorPMT
```

Comment:
Not sure yet how to connect the metadata to the actual traces, need to setup a meeting with them to ask.